package.json

```diff
@@ -26,7 +24,11 @@
     "sass": "sass --watch src/styles/scss:src/styles/css",
     "precommit": "lint-staged",
     "analyze": "source-map-explorer build/static/js/main.*",
-    "storybook": "start-storybook -p 9001 -c .storybook"
+    "storybook": "start-storybook -p 9001 -c .storybook",
+    "lingui-init": "lingui init",
```

Comment:
This task is not necessary, please remove it. The `lingui  init` should only be ran once when installing the project, you have done that already, no need for other to do it.